backend/test/services/LibraryService.test.ts

```diff
@@ -180,6 +180,74 @@ describe('LibraryService', () => {
     });
   });
 
+  describe('create', () => {
+    it('returns 201 status with populated location when provided correct headers and a FHIR Library', async () => {
+      await supertest(server.app)
+        .post('/4_0_1/Library')
+        .send({ resourceType: 'Library', status: 'draft' })
+        .set('content-type', 'application/json+fhir')
+        .expect(201)
+        .then(response => {
+          expect(response.headers.location).toBeDefined();
+        });
+    });
+  });
+
+  describe('update', () => {
+    it('returns 200 when provided correct headers and a FHIR Library whose id is in the database', async () => {
+      await createTestResource(
+        { resourceType: 'Library', type: { coding: [{ code: 'logic-library' }] }, id: 'exampleId', status: 'draft' },
+        'Library'
+      );
```

Comment:
I would put this resource creation into a `beforeAll` call in this describe block. Not sure there's anything wrong with the way you've done it here, but that's been our standard for creating test resources in the past.